modules/investigating-kernel-crashes.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operating-system-issues.adoc
+
+[id="investigating-kernel-crashes"]
+= Investigating kernel crashes
+
+== Enabling kdump
+
+`kdump` (included in `kexec-tools`) is a service providing a crash dumping mechanism. The service enables you to save the contents of the system’s memory for later analysis.
+
+{op-system} ships with `kexec-tools`, but manual configuration is required to enable `kdump`.
+
+The following are the basic steps needed to enable `kdump` on {op-system}.
+
+.  Memory must be reserved for the crash kernel during booting of the first kernel. Kernel arguments can be provided like this:
```

Comment:
Suggested edit: 

s/Kernel arguments can be provided like this:/Provide kernel arguments by entering the following command:

Reply:
Also above the first bullet, suggested edit:

s/The following are the basic steps needed to enable `kdump` on {op-system}./The following steps are needed to enable `kdump` on {op-system}.